integration/go.mod

```diff
@@ -0,0 +1,24 @@
+module github.com/hyperledger/fabric-chaincode-evm/integration
+
+replace github.com/hyperledger/fabric-chaincode-evm/fab3 => ../fab3
+
+replace github.com/hyperledger/fabric-chaincode-evm/evmcc => ../evmcc
```

Comment:
I was going to ask if all the replaces should be together, but I see that these are the local refs to the other repos.

Reply:
moved this down to both so all replaces are together.

Reply:
yay